tests/integration/test_json_schema_generation.py

```diff
@@ -0,0 +1,71 @@
+import os
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+
+from schematic.configuration.configuration import Configuration
+from schematic.schemas.commands import schema
+from tests.conftest import Helpers
+
+
+@pytest.fixture
+def runner() -> CliRunner:
+    """Fixture for invoking command-line interfaces."""
+
+    return CliRunner()
+
+
+@pytest.fixture
+def data_model_jsonld(helpers):
+    data_model_jsonld = helpers.get_data_path("example.model.jsonld")
+    yield data_model_jsonld
+
+
+class TestSchemaCli:
+    # TODO: change url target to develop branch
+    @pytest.mark.parametrize(
+        "data_model_location",
+        [
+            "example.model.jsonld",
+            "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/refs/heads/schematic-276-jsonschema-cli/tests/data/example.model.jsonld",
```

Comment:
Here, the json model url contains schematic-276-jsonschema-cli which is the branch name. Can we switch it to the main branch?

Reply:
I would use a permanent link instead of pointing to main or feature branch.